applications/dot_marketplace-Phase3.md

```diff
@@ -0,0 +1,222 @@
+| 1a | Disapprove Milestone  | In the case of a customer not being satisfied with the work submitted by the service provider (worker). A set of jurors is shortlisted (court summon) to resolve the dispute and pass a verdict. |
+| 1b | Disapprove Rating | The customer or the service provider, once they have received their rating for a particular milestone and are not satisfied with it. |
+| 1c | General Dispute | A general dispute function for cases that do not fall under the categories mentioned in 1a and 1b. |
+| 2 | Voting module | Each juror can review the dispute and cast their vote, which also includes their rating for both the customer and the worker. After two days, all the juror votes are counted, and a winner is identified. |
+| 3 | Frontend App  | Supporting frontend UI to test the aforementioned functionality. |
+
+### **Milestone 3**
+
+* **Estimated duration:** 3.0 weeks
+* **FTE:** 1
+* **PTE:** 2
+* **Costs:** 7975 USD
+
+
+The main deliverables in this milestone are to use decentralized IPFS based storages to store all the files realated to tasks & advanced search. A file server integrated to nft storage will provided, using rocket framework & the search feature will be an update to the makerplace pallet. The skeletal UI will also be updated to showcase all the new features in Phase3.
+
+| Sr no. | Deliverable | Description |
+| --- | --- | --- |
+| 0a | License | Apache 2.0 |
+| 0b | Documentation | We will provide both inline documentation of the code and a tutorial that explains how a user can use DOT Marketplace and understand the flow of tasking pallet. |
+| 0c | Testing Guide | Functions will be covered by unit tests, the documentation will describe how to run these tests. We will also provide scripts to help deploy, run and test the build. |
+| 0d | Docker Image | Docker image of the build |
+| 1 | Decentralized Stroage  | All tasks related docs will be stored on a decentralized IPFS platform. |
+| 2 | Advanced Search  | Search based on task progress, tags, tasks or milestone id's. |
+| 3 | Frontend App  | Supporting frontend UI to test the aforementioned functionality. |
+| 4 | Website  | Dedicated one-page website for Dot Marketplace. |
+| 5 | Article | Website article showing motivation behind phase 3 of dot marketplace and how to make the best use of it. |
+
+### **Additional Project Details**
+
+* Technology stack being used
+  * Rust, Substrate, React, Python, centralized cloud storage
+
+### **Future Plans**
+
+This is the last phase in our current roadmap. Post this we would focus on partnerships with chains on the dotsama ecosystem for integrating DotMarketplace as their native bounty management tool (this work has already started). If future, if the traction is great, we could create a fresh proposal for an excellent UI or integrate DotMarketplace within PolkaJS Apps itself with native support for multiple tokens besides DOT.
```

Comment:
> this work has already started

Can you update us on your progress here? In general, I agree with [Sebastians comment](https://github.com/w3f/Grants-Program/pull/1205#issuecomment-1283033759) that it would be nice to see some indication here that there is interest in the community to use your product.

Reply:
I'm back from my vacation.  
Can we take this up over chat? 
Would be happy to share messages exchanged with three projects. 
As well as show the UI, I mentioned earlier. 
Thank you.

Reply:
@abeoye Unless you already talked to @takahser, can you send them to grants@web3.foundation? Otherwise, let me know your matrix handle and I'll create a room for you to post them in.

Reply:
Adding @abeoye's matrix handle [here](https://matrix.to/#/@amit.wow:matrix.org)

Reply:
Hello @semuelle @takahser @Noc2 @alxs, 
Invited you to this [matrix room](https://matrix.to/#/!tQIEODncpjnXbyQfqc:matrix.org?via=matrix.org). 
Thanks!